src/Plugin/VoteUpDownWidgetBase.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Drupal\vud\Plugin;
+
+use Drupal\Component\Plugin\PluginBase;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+
+/**
+ * Base class for Vote Up/Down Widget plugins.
+ */
+abstract class VoteUpDownWidgetBase extends PluginBase implements VoteUpDownWidgetInterface {
```

Comment:
I remember a couple of extra data members were added into the base class after one of my comments, why did you remove them?
Original comment about this on https://github.com/marvil07/vote_up_down/pull/3#discussion_r122050230